frontends/precinct-scanner/src/utils/save_cvr_export_to_usb.test.ts

```diff
@@ -73,9 +82,8 @@ test('calls kiosk saveAs when opening file picker dialog', async () => {
     isTestMode: false,
     openFilePickerDialog: true,
   });
-  expect(window.kiosk.saveAs).toHaveBeenCalledWith({
-    defaultPath:
-      'fake mount point/cast-vote-records/sample-county_example-primary-election_0dabcacc5d/machine_0003__0_ballots__2020-10-31_00-00-00.jsonl',
+  expect(kiosk.saveAs).toHaveBeenCalledWith({
+    defaultPath: 'machine_0003__0_ballots__2020-10-31_00-00-00.jsonl',
```

Comment:
the default "Save As" path for CVRs is no longer within a specific directory, but just at the root of the USB drive. seems unimportant